docs/recipes/static-asset-revisioning.md

```diff
@@ -0,0 +1,33 @@
+# Static asset revisioning
+
+Static asset revisioning by appending content hash to filenames. Make sure to set the files to [never expire](http://developer.yahoo.com/performance/rules.html#expires) for this to have an effect.
+
+```sh
+npm install --save-dev gulp gulp-rev gulp-rev-rewrite gulp-rev-delete-original
```

Comment:
I don't want to recommend `gulp-rev-delete-original` because it's not maintained.

Reply:
Good